egs/mgb2/asr/local/score_mbr.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
We do not have to include this.
Please remove it.

Reply:
done